apps/web/frontend/components/userLabel.tsx

```diff
@@ -2,16 +2,16 @@ import Image from 'next/image'
 
 interface UserLabelProps {
   name: string
-  duration: number
+  duration?: number
```

Comment:
Suggestion: what about displaying the duration?

Reply:
Well, I believe that won't fit here because we don't query a single project and having the duration of a user for all projects won't be an important info. I will still raise it on our daily though.